tasks/conf.yml

```diff
@@ -44,6 +44,12 @@
 - name: Set --http-port option
   template: src=http_port.j2 dest=/etc/marathon/conf/http_port
 
+- name: Set additional options
+  template: src=custom_option.j2 dest=/etc/marathon/conf/{{ item.name }}
```

Comment:
ps2: I also agree we should more use the 100% json notation (like in your example above, with 1 line per attribute)
Should we do a big refactoring (after merging all acceptable PRs) !?